packages/cms-data-sync/src/utils/entries.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-console */
+import { Environment, Entry } from 'contentful-management';
+
+export const clearContentfulEntries = async (
+  contentfulEnvironment: Environment,
+  contentType: string,
+) => {
+  const entries = await contentfulEnvironment.getEntries({
+    content_type: contentType,
+  });
+  console.log(
+    `Cleaning Contentful Entries from content-type ${contentType}...`,
+  );
+
+  const unpublishPromises: Promise<Entry>[] = [];
+  entries.items.forEach(async (entry) => {
```

Comment:
again this could be a simple map

Reply:
committed here https://github.com/yldio/asap-hub/pull/2290/commits/fddabef2153bb8586f2cabcaee3b8cccf89e44bf